sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -238,8 +238,8 @@ gnrc_pktsnip_t *gnrc_ndp_opt_rdnss_build(uint32_t ltime, ipv6_addr_t *addrs,
         rdnss_opt->resv.u16 = 0;
         rdnss_opt->ltime = byteorder_htonl(ltime);
         for (unsigned i = 0; i < addrs_num; i++) {
-            memcpy(&rdnss_opt->addrs[i], &addrs[i],
-                   sizeof(rdnss_opt->addrs[i]));
+            memcpy(&(ndp_opt_rdnss_addrs_ptr(rdnss_opt)[i]), &addrs[i],
```

Comment:
Can you rather use a temporary variable to appoint the result of `ndp_opt_rdnss_addrs_ptr(rdnss_opt)`, please? I think that would make the code a bit more readable.